CONTRIBUTORS.txt

```diff
@@ -154,6 +154,9 @@ In chronological order:
 * Jon Wayne Parrott <jonwayne@google.com>
   * App Engine environment tests.
 
+* James Atherfold <jlatherfold@hotmail.com>
+  * Bugfixes.
```

Comment:
Can you be a biiit more descriptive? :D Maybe "Fix bug with reusing connections after error"?

(I know there's another bugfixes above and I've regretted it lol.)